server/fishtest/views.py

```diff
@@ -1005,6 +1007,12 @@ def tests_modify(request):
         run["args"]["priority"] = int(request.POST["priority"])
         run["args"]["throughput"] = int(request.POST["throughput"])
         run["args"]["auto_purge"] = True if request.POST.get("auto_purge") else False
+        if (
+            is_same_user(request, run)
+            and "msg-new" in request.POST
+            and request.POST["msg-new"].strip() != ""
+        ):
+            run["args"]["msg_new"] = request.POST["msg-new"].strip()
```

Comment:
```diff
- run["args"]["msg_new"] = request.POST["msg-new"].strip()
+ run["args"]["info"] = request.POST["msg-new"].strip()
```

Reply:
> ```diff
> - run["args"]["msg_new"] = request.POST["msg-new"].strip()
> + run["args"]["info"] = request.POST["msg-new"].strip()
> ```

not really try now at DEV

Reply:
Look at DEV, you are changing the wrong information.

Reply:
"msg_base" and "msg_new" are the commit messages